prijateli_tree/app/schemas.py

```diff
@@ -8,6 +8,19 @@
 from pydantic import BaseModel
 
 
+class GameCreate(BaseModel):
+    created_by: int
+    game_type_id: int
+    rounds: int
+    practice: bool
+
+
+class PlayerCreate(BaseModel):
+    user_id: int
+    position: int
+    name_hidden: bool
```

Comment:
Should these be inheriting from other models?

Reply:
I dont think so! I created them for pydantic to check the parameters we send through the API are correct